lib/pbio/drv/bluetooth/bluetooth_stm32_bluenrg.c

```diff
@@ -472,12 +470,18 @@ static PT_THREAD(scan_and_connect_task(struct pt *pt, pbio_task_t *task)) {
     PT_WAIT_UNTIL(pt, hci_command_status);
     context->status = aci_gatt_disc_charac_by_uuid_end();
 
-    PT_WAIT_UNTIL(pt, {
-        if (task->cancel) {
-            goto cancel_disconnect;
-        }
-        remote_lwp3_char_handle;
-    });
+    // HACK: Chararacteristics of LEGO Mario are not properly found by GATT_DiscCharsByUUID(). 
+    // remote_lwp3_char_handle for mario is hard coded for now 
+    if(context->hub_kind == LWP3_HUB_KIND_MARIO){
```

Comment:
We use an automatic code format tool to try to keep things somewhat consistent. See https://github.com/pybricks/pybricks-micropython/pull/64/checks?check_run_id=3281639134

There should be spaces before and after the `()` here. 

There are also trailing spaces at the end of the line in a few other places.

Reply:
> See inline comments for some small things that need to be fixed before we merge.

Thanks for the feedback! I will apply the required updates.